pom.xml

```diff
@@ -150,15 +150,16 @@
 					<version>3.2.0</version>
 				</plugin>
 
-				<plugin>
+					<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-checkstyle-plugin</artifactId>
-					<version>${checkstyle.plugin.version}</version>
```

Comment:
r u using mvn >= 3.3.9 ?  At a minimum, u need mvn 3.3.9 and Java 8 for OpenNLP. This PR now has broken a working Travis Build.